recipes/cutensor/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: cutensor
+  version: {{ version }}
+
+{% set cuda_compiler_version = cuda_compiler_version or 'undefined' %}
+{% if cuda_compiler_version != "undefined" %}
+  {% if cuda_compiler_version != "None" %}
+    {% set ver_str = cuda_compiler_version.replace(".", "") %}
+  {% else %}
+    {% set ver_str = "" %}
+  {% endif %}
+{% endif %}
+
+{% set suffix = "x86_64-linux-gnu" %}       # [linux64]
+{% set suffix = "powerpc64le-linux-gnu" %}  # [ppc64le]
+{% set suffix = "x86_64-w64-mingw32" %}     # [win64]
+
+source:
+  url: https://github.com/JuliaBinaryWrappers/CUTENSOR_CUDA{{ ver_str }}_jll.jl/releases/download/CUTENSOR_CUDA{{ ver_str }}-v{{ version }}%2B0/CUTENSOR_CUDA{{ ver_str }}.v{{ version }}.{{ suffix }}.tar.gz  # [cuda_compiler_version in ("10.1", "10.2", "11.0")]
```

Comment:
I'm going to follow up internally regarding a machine downloadable version before we use this.

Reply:
Thanks for digging out the link, @kkraus14! Marking as resolved.